UpdateDependencies.ps1

```diff
@@ -0,0 +1,52 @@
+﻿#
```

Comment:
Nit: I would have preferred the ps1 and the version changes to be separate commits within this PR.  It makes cherry picking easier if you don't have to take both.  No need to change for this PR.